test/functional/wallet_listreceivedby.py

```diff
@@ -103,6 +104,27 @@ def test_listreceivedbyaddress(self):
         res = self.nodes[1].listreceivedbyaddress(0, True, True, other_addr)
         assert_equal(len(res), 0)
 
+    def test_listreceivedby(self):
```

Comment:
In 62880a9bb167bc0569175d597a865e6baccce71b "test: add coverage for `listreceivedby*` no "send" purpose addrs return"

nit: `s/test_listreceivedby/test_listreceivedby_excludes_send`